test/tests/functional/pbs_user_reliability.py

```diff
@@ -80,15 +80,17 @@ def test_create_resv_from_job_using_runjob_hook(self):
         jid = self.server.submit(job)
         self.server.expect(JOB, {ATTR_state: 'R'}, jid)
 
-        rid = 'R' + str(int(jid.split(".")[0]) + 1)
-        a = {ATTR_queue: rid}
-        self.server.expect(JOB, a, id=jid)
+        a = {ATTR_job: jid}
+        rid = self.server.status(RESV, a)[0]['id'].split(".")[0]
 
         a = {ATTR_job: jid, 'reserve_state': "RESV_RUNNING",
              'Resource_List.select': '1:ncpus=3',
              'Resource_List.walltime': 9999}
         self.server.expect(RESV, a, id=rid)
 
+        a = {ATTR_queue: rid}
```

Comment:
When checking for the state of a reservation do (MATCH_RE, 'RESV_RUNNING|2)' (here and other places)